dom.bs

```diff
@@ -1864,6 +1868,17 @@ these steps:
   </ol>
 </ol>
 
+<p>The
+<dfn method for=AbortSignal lt="aborted()" id="dom-abortsignal-aborted-static">
+<code>AbortSignal.aborted()</code></dfn> steps are:
```

Comment:
I added the `id` attribute because `make` was complaining about a duplicate id `dom-abortsignal-aborted` ID (because of the attribute). Also, when I remove the `lt` attribute, `make` complains:

```
Error running preprocessor, returned code: 2.
FATAL ERROR: Function/method names can only contain alphanums, underscores, dashes, or []. Got 'AbortSignal.aborted()'.
 ✘  Did not generate, due to fatal errors
```

Reply:
You want to remove `AbortSignal.`. We just use the word "static". I guess the ID issue might be real though, but could go away if we switch to `abort()`.

Reply:
Ok, so change to `the static aborted() steps are:` ?

And switching to `AbortSignal.abort()` is just fine.

Reply:
Ok, done! Renamed to `AbortSignal.abort()` and updated the text to allow the `lt` and `id` to be removed!